[docs] Batch14 changes to hll functions (#4)
edgar2020/druid: docs/querying/sql-scalar.md

|Function|Notes|
|--------|-----|

> |`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns a distinct count estimate from an HLL sketch. `expr` must be an HLL sketch. To round the estimate, set `round` to true. Otherwise, `round` defaults to false.|

Reviewer: ```suggestion
|`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns a distinct count estimate from a HLL sketch. `expr` must be a HLL sketch. To round the estimate, set `round` to true. Otherwise, `round` defaults to false.|
```

> |`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns the distinct count estimate from an HLL sketch. `expr` must return an HLL sketch. The optional `round` boolean argument rounds the estimate if set to `true`, with a default of `false`.|

Reviewer: ```suggestion
|`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns a distinct count estimate from an HLL sketch. `expr` must be an HLL sketch. To round the estimate, set `round` to true. Otherwise, `round` defaults to false.|
```

> |`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr[, numStdDev])`|Returns a distinct count estimate and error bounds from an HLL sketch. `expr` must be an HLL sketch. `numStdDev` argument specifies the number of standard deviations of the bounds. `numStdDev` must be `1`, `2`, or `3`. |

Reviewer: ```suggestion
|`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr[, numStdDev])`|Returns a distinct count estimate and error bounds from a HLL sketch. `expr` must be a HLL sketch. `numStdDev` argument specifies the number of standard deviations of the bounds. `numStdDev` must be `1`, `2`, or `3`. |
```

> |`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr[, numStdDev])`|Returns the distinct count estimate and error bounds from an HLL sketch. `expr` must return an HLL sketch. An optional `numStdDev` argument specifies the number of standard deviations of the bounds. `numStdDev` must be `1`, `2`, or `3`. |

Reviewer: ```suggestion
|`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr[, numStdDev])`|Returns a distinct count estimate and error bounds from an HLL sketch. `expr` must be an HLL sketch. `numStdDev` argument specifies the number of standard deviations of the bounds. `numStdDev` must be `1`, `2`, or `3`. |
```
